nannou/src/draw/renderer/mod.rs

```diff
@@ -413,7 +417,7 @@ impl Renderer {
         let uniforms = create_uniforms(output_attachment_size, output_scale_factor);
         let uniforms_bytes = uniforms_as_bytes(&uniforms);
         let usage = wgpu::BufferUsage::UNIFORM | wgpu::BufferUsage::COPY_DST;
-        let uniform_buffer = device.create_buffer_with_data(uniforms_bytes, usage);
+        let uniform_buffer = device.create_buffer(uniforms_bytes, usage);
```

Comment:
The equivalent in wgpu 0.6 to `create_buffer_with_data()` is [`create_buffer_init()`](https://docs.rs/wgpu/0.6.0/wgpu/util/trait.DeviceExt.html#tymethod.create_buffer_init).

Reply:
[8ed73cb ](https://github.com/nannou-org/nannou/pull/655/commits/8ed73cb3fc891e1901080edc115f37369a4c1d0d)